packages/compass-databases-navigation/src/icon-button.tsx

```diff
@@ -78,7 +79,7 @@ export const IconButtonSmall = forwardRef<
       // TODO: we should probably explicitly set our own colors given that we're
       // setting all the colours around it. Rather than rely on leafygreen's
       // default dark or light hover background colour.
-      darkMode={darkMode}
+      darkMode={theme === Theme.Dark}
```

Comment:
IIRC all leafygreen components that we re-export from compass-components are already connected to theme provider, so I think there should be no need to pass this here explicitly